### DISCUSSION TITLE
Doesn't collect statistics

### DISCUSSION THREAD

**Discussion by strikekormoran** [pretrehr/Sports-betting#6] [General]
There are no results after parsing the sites.
Tell me what the problem is. I attach the error screen.
![6](https://user-images.githubusercontent.com/76999687/104295162-e9187100-54e1-11eb-99e1-7b95a0d5d58d.png)

**Comment by pretrehr** [comment]
Hi ! Thanks for your message. I'll check that and keep you up to date.

**Comment by pretrehr** [comment]
For anyone who is facing the same kind of problem, it was solved in issue #7.
